### PR TITLE
Fix apps stack not deleted on AWS GovCloud

### PR DESCRIPTION
**Pull request by yunjiang29**

### CODE REVIEW

File: ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh [hunk 1]
```diff
@@ -330,6 +330,7 @@ EOF
   fi
 
   if [ "${CLUSTER_TYPE}" == "aws-usgov" ]; then
+    echo ${APPS_DNS_STACK_NAME} >> "${SHARED_DIR}/to_be_removed_cf_stack_list"
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh:333]: Can we replace line 310 with this line?

Comment (yunjiang29) [reply ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh]: fixed

File: ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh [hunk 2]
```diff
@@ -307,7 +307,7 @@ EOF
   fi
 
   APPS_DNS_STACK_NAME="${CLUSTER_NAME}-apps-dns"
-  echo ${APPS_DNS_STACK_NAME} > "${SHARED_DIR}/apps_dns_stack_name"
+  echo ${APPS_DNS_STACK_NAME} > "${SHARED_DIR}/to_be_removed_cf_stack_list"
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh:310]: seem like should be `>>` here?

Comment (yunjiang29) [reply ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh:310]: Thanks!

**Pull request merged**